### DISCUSSION TITLE
How to Type into iframe Rich Text Editor Text Box?

### DISCUSSION THREAD

**Discussion by AI-Based-Businesses** [AutomaApp/automa#1397] [Q&A]
I'm having the same problem as mentioned here: https://github.com/AutomaApp/automa/discussions/194
Axiom successfully selects the text box using the following selector: "div:nth-of-type(2) > div > div > iframe"
Has anyone found a successful selector for Automa?

**Comment by Delusionnnn** [comment]
Switch frame block doesn't work either?

**Reply by AI-Based-Businesses** [reply]
Yeah, I can't get any block to select this text box.
